src/cmd.ts

```diff
@@ -63,6 +63,11 @@ function createCommandWithSharedOptions(name, description) {
 			"Torznab urls with apikey included (separated by spaces)",
 			fallback(fileConfig.torznab)
 		)
+		.option(
+			"-d, --data-dir <dir>",
```

Comment:
you can refer to the torznab syntax right above this to support an array of datadirs as space-delimited option arguments.

Reply:
Will do in the next round of post-review coding.